File: src/main/java/net/hectus/neobb/turn/default_game/structure/glass_wall/TPinkGlassWall.java
package net.hectus.neobb.turn.default_game.structure.glass_wall;

import net.hectus.neobb.player.NeoPlayer;
import net.hectus.neobb.structure.PlacedStructure;
import net.hectus.neobb.turn.default_game.attributes.clazz.SupernaturalClazz;

public class TPinkGlassWall extends GlassWallTurn implements SupernaturalClazz {
    public TPinkGlassWall(NeoPlayer player) { super(player); }
    public TPinkGlassWall(PlacedStructure data, NeoPlayer player) { super(data, player); }

    @Override
    public String color() {
        return "pink";
    }
}

File: src/main/kotlin/net/hectus/neobb/turn/default_game/structure/glass_wall/TPinkGlassWall.kt
package net.hectus.neobb.turn.default_game.structure.glass_wall

import com.marcpg.libpg.storing.Cord
import net.hectus.neobb.player.NeoPlayer
import net.hectus.neobb.structure.PlacedStructure
import net.hectus.neobb.structure.StaticStructure
import net.hectus.neobb.structure.StaticStructures
import net.hectus.neobb.turn.default_game.attribute.clazz.SupernaturalClazz

class TPinkGlassWall(data: PlacedStructure?, cord: Cord?, player: NeoPlayer?) : GlassWallTurn(data, cord, player), SupernaturalClazz {
    override val staticStructure: StaticStructure = StaticStructures.Default.GlassWall.PINK
}
